projects/media-viewer/src/lib/viewers/image-viewer/image-viewer.component.ts

```diff
@@ -21,41 +21,24 @@ import {ToolbarEventsService} from '../../shared/toolbar-events.service';
     templateUrl: './image-viewer.component.html',
     styleUrls: ['./image-viewer.component.scss'],
 })
-export class ImageViewerComponent implements OnInit, OnDestroy, OnChanges {
+export class ImageViewerComponent implements OnChanges {
 
   @Input() url: string;
   @Input() downloadFileName: string;
   @Input() zoomValue: Subject<ZoomValue>;
   @Input() annotationSet: AnnotationSet | null;
 
-  drawMode = false;
   errorMessage: string;
 
   @ViewChild('img') img: ElementRef;
   rotation = 0;
   zoom = 1;
 
-  // local array of any subscriptions so that we can tidy them up later
-  private subscriptions: Subscription[] = [];
-
   constructor(
     private readonly printService: PrintService,
-    private readonly toolbarEventsService: ToolbarEventsService
+    public readonly toolbarEventsService: ToolbarEventsService
```

Comment:
the suggested view change in the comment above would mean that this can stay private